src/index.ts

```diff
@@ -112,13 +112,11 @@ prog
   .action(async (pkg: string, opts: any) => {
     console.log(
       chalk.blue(`
-::::::::::: ::::::::  :::::::::  :::    :::
-    :+:    :+:    :+: :+:    :+: :+:    :+:
-    +:+    +:+        +:+    +:+  +:+  +:+
-    +#+    +#++:++#++ +#+    +:+   +#++:+
-    +#+           +#+ +#+    +#+  +#+  +#+
-    #+#    #+#    #+# #+#    #+# #+#    #+#
-    ###     ########  #########  ###    ###
+      ██    ██ ████████ ███████ ██████  ██   ██ 
+      ██    ██    ██    ██      ██   ██  ██ ██  
+      ██    ██    ██    ███████ ██   ██   ███   
+      ██    ██    ██         ██ ██   ██  ██ ██  
+       ██████     ██    ███████ ██████  ██   ██ 
```

Comment:
there is problem in this content

Reply:
![image](https://user-images.githubusercontent.com/5729248/143670398-0ac4bac0-2a15-4645-a1e4-e40086b8482e.png)
It's ok btw